demisto_sdk/commands/content_graph/parsers/repository.py

```diff
@@ -42,14 +45,25 @@ def parse(
         try:
             logger.debug("Parsing packs...")
             with multiprocessing.Pool(processes=cpu_count()) as pool:
-                for pack in pool.imap_unordered(PackParser, packs_to_parse):
-                    self.packs.append(pack)
-                    if progress_bar:
-                        progress_bar.update(1)
+                for pack in pool.imap_unordered(
+                    RepositoryParser.parse_pack, packs_to_parse
+                ):
+                    if pack:
+                        self.packs.append(pack)
+                        if progress_bar:
+                            progress_bar.update(1)
         except Exception:
             logger.error(traceback.format_exc())
             raise
 
+    @staticmethod
+    def parse_pack(pack_path: Path) -> Optional[PackParser]:
+        try:
+            return PackParser(pack_path)
+        except NotAContentItemException:
+            logger.warning(f"Pack {pack_path.name} is not a valid pack. Skipping")
```

Comment:
```suggestion
            logger.error(f"Pack {pack_path.name} is not a valid pack. Skipping")
```

maybe its better to log an error in case we can't parse a pack?